src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const cost = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+  let vocationCost = 0;
```

Comment:
The variable name 'vocationCost' may be a typo or incorrect term. It should be 'vacationCost' to accurately reflect its purpose.